app/views/steps/caution/result/shared/_row.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-summary-list__row">
+  <dt class="govuk-summary-list__key">
+    <%=t "caution_results_html.#{row.question.to_s}.question" %>
```

Comment:
I think it is not necessary to do `#to_s` here, as the interpolation of a symbol will cast a string. Can you double check? (same thing a few lines below).